main.py

```diff
@@ -2,122 +2,181 @@
 
 import argparse
 import math
+import sys
 
 "Performs addition"
-def add(a,b):
-    val = a+b
+
+
+def add(a, b):
+    val = a + b
     return val
 
 "Performs subtraction"
-def sub(a,b):
-    val = a-b
+
+
+def sub(a, b):
+    val = a - b
     return val
 
 "Performs division"
-def div(a,b):
-    val = a/b
+
+
+def div(a, b):
+    try:
+        val = a / b
+        if b == 0:
+            raise
+    except:
+        print("Cannot divide by Zero !")
+        sys.exit(0)
     return val
 
 "Performs multiplication"
-def multi(a,b):
-    val = a*b
+
+
+def multi(a, b):
+    val = a * b
     return val
 
 "Performs percentage"
+
+
 def perc(a):
-    val = a/100
+    val = a / 100
     return val
 
 "Performs factorial"
+
+
 def fact(a):
     return math.factorial(a)
 
 "Performs sqaure root"
+
+
 def squarert(a):
     return math.sqrt(a)
 
 "Performs power operation"
-def power(a,b):
-    return math.pow(a,b)
+
+
+def power(a, b):
+    return math.pow(a, b)
 
 "Performs base-10 logarithm"
+
+
 def log(a):
     return math.log10(a)
 
 "Performs sine operation"
+
+
 def sin(a):
     return math.sin(math.radians(a))
 
 "Performs cosine operation"
+
+
 def cos(a):
     return math.cos(math.radians(a))
 
 "Performs tangent operation"
+
+
 def tan(a):
     return math.tan(math.radians(a))
 
 "Converts angle a from radians to degrees"
+
+
 def deg(a):
     return math.degrees(int(a))
 
 "Converts angle a from degrees to radians"
+
+
 def rad(a):
     return math.radians(a)
 
 "The Main function"
+
+
 def main():
-    parser = argparse.ArgumentParser(description = "A Command Line Calculator", epilog = "Enter two numbers for addition,subtraction and so on....\n Enter only one number for percentage calculation,factorial and so on....")
-	
+    parser = argparse.ArgumentParser(description="A Command Line Calculator",
+                                     epilog="Enter two numbers for addition,subtraction and so on....\n Enter only one number for percentage calculation,factorial and so on....")
```

Comment:
It would be nice if we can follow PEP8 rule. Could you break the text in couple of lines?